server/single-server/src/main/resources/logback.xml

```diff
@@ -1,12 +1,9 @@
 <configuration>
-    <!-- reference: https://github.com/slick/slick/blob/master/common-test-resources/logback.xml -->
     <logger name="slick" level="OFF" />
     <logger name="com.zaxxer.hikari" level="OFF" />
-    <!--<logger name="slick.jdbc.JdbcBackend.statement" level="DEBUG" />-->
-    <!--<logger name="slick.jdbc.StatementInvoker.result" level="DEBUG" />-->
-    <!--<logger name="slick.jdbc.JdbcBackend.benchmark" level="DEBUG" />-->
+    <logger name="cool.graph" level="INFO" />
 
-    <logger name="cool.graph" level="DEBUG" />
+    <statusListener class="ch.qos.logback.core.status.NopStatusListener" />
```

Comment:
what does that do?

Reply:
That supresses the logback status messages themselves, e.g. lines like `20:46:10,657 |-INFO in ch.qos.logback.classic.LoggerContext[default] - ...`